dojo/forms.py

```diff
@@ -3177,19 +3217,28 @@ def __init__(self, *args, **kwargs):
         else:
             questionnaire_permissions = []
 
-        permission_fields_1 = [
-            Permission_Helper(name='development environment', app='dojo', view=True, add=True, change=True, delete=True),
-            Permission_Helper(name='finding template', app='dojo', view=True, add=True, change=True, delete=True),
-            Permission_Helper(name='group', app='auth', view=True, add=True),
-            Permission_Helper(name='permission', app='auth', change=True)
+        permission_fields_4 = [
+            Permission_Helper(name='regulation', app='dojo', add=True, change=True, delete=True),
         ]
-        permission_fields_2 = [
-            Permission_Helper(name='test type', app='dojo', view=True, add=True, change=True),
+
+        if get_system_setting('enable_rules_framework'):
+            rules_permissions = [
+                Permission_Helper(name='rule', app='auth', view=True, add=True, change=True, delete=True),
+            ]
+        else:
+            rules_permissions = []
+
+        permission_fields_5 = [
+            Permission_Helper(name='test type', app='dojo', add=True, change=True),
+            Permission_Helper(name='tool configuration', app='dojo', view=True, add=True, change=True, delete=True),
             Permission_Helper(name='tool type', app='dojo', view=True, add=True, change=True, delete=True),
             Permission_Helper(name='user', app='auth', view=True, add=True, change=True, delete=True),
         ]
 
-        self.permission_fields = permission_fields_1 + questionnaire_permissions + permission_fields_2
+        self.permission_fields = permission_fields_1 + github_permission + google_sheet_permission + \
+                                 permission_fields_2 + jira_permissions + permission_fields_3 + \
+                                 questionnaire_permissions + permission_fields_4 + rules_permissions + \
+                                 permission_fields_5
```

Comment:
Can we append to a single large list of `Permissions_Helper` objects rather than creating many small lists and then adding them all together at the bottom? Would look much cleaner

Reply:
@Maffooch Does this look better now?